syncs/shardedmap_test.go

```diff
@@ -41,4 +41,41 @@ func TestShardedMap(t *testing.T) {
 	if g, w := m.Len(), 0; g != w {
 		t.Errorf("got Len %v; want %v", g, w)
 	}
+
+	// Mutation adding an entry.
+	if v := m.Mutate(1, func(was string, ok bool) (string, bool) {
```

Comment:
Is it worthwhile also verifying that the implementation made the desired change in the map?

Reply:
Ah yeah, so it does.